### PR TITLE
A new section about creating SP-independent links

### PR DESCRIPTION
**Pull request by dariavladykina**
Added a new section on creating SP-independent links per issue #200.

### CODE REVIEW

**Review by janajaeger — APPROVED** [review]
I like it very much. Thumbs up from me!

File: xml/docu_styleguide.structure.xml [hunk 1]
```diff
@@ -923,6 +923,128 @@ will be displayed in &lt;emphasis role="bold"&gt;bold&lt;/emphasis&gt;</screen>
...
+   <para>
+     You want to support SEO where the most prominent SP is more important 
+     than previous, older SPs
+    </para>
+   </listitem>
+  </itemizedlist>
+  <para>
+   Make sure to follow this syntax:
+  </para>
+  <screen>&lt;documentation.suse.com/<replaceable>&lt;PRODUCT></replaceable>[/<replaceable>&lt;MAJOR_RELEASE></replaceable>]/<replaceable>&lt;DEEP_LINK>&gt;</replaceable></screen>
```
Comment (tomschr) [inline xml/docu_styleguide.structure.xml:958]: I think the first `&lt;` should be removed:

```suggestion
  <screen>documentation.suse.com/<replaceable>&lt;PRODUCT></replaceable>[/<replaceable>&lt;MAJOR_RELEASE></replaceable>]/<replaceable>&lt;DEEP_LINK>&gt;</replaceable></screen>
```

File: xml/docu_styleguide.structure.xml [hunk 2]
```diff
@@ -923,6 +923,128 @@ will be displayed in &lt;emphasis role="bold"&gt;bold&lt;/emphasis&gt;</screen>
...
+  <itemizedlist>
+    <listitem>
+    <para>
+     <replaceable>PRODUCT</replaceable>: the abbreviation of the product, like <quote>sle</quote> for &sle;,
+     <quote>sle-ha</quote> for &sle; Server High Availability Extension, etc.
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     <replaceable>MAJOR_RELEASE</replaceable>: an optional major release like 12 or 15
```
Comment (tomschr) [inline xml/docu_styleguide.structure.xml:971]: Maybe add an additional information what happens, when you don't provide a major release:

```suggestion
     <replaceable>MAJOR_RELEASE</replaceable>: an optional major release like 12 or 15. If you omit it,
     your link will be redirected to use the most recent release.
```

File: xml/docu_styleguide.structure.xml [hunk 3]
```diff
@@ -923,6 +923,128 @@ will be displayed in &lt;emphasis role="bold"&gt;bold&lt;/emphasis&gt;</screen>
...
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     <replaceable>DEEP_LINK</replaceable>: the link that points to a specific chapter or section within a book
+    </para>
+   </listitem>
+  </itemizedlist>
+  <para>
+   Make sure you use <quote>html</quote> and not <quote>single-html</quote>. This
```
Comment (tomschr) [inline xml/docu_styleguide.structure.xml:981]: I would prefer `literal` here as it's a part of a URL:

```suggestion
   Make sure you use <literal>html</literal> and not <literal>single-html</literal>. This 
```

File: xml/docu_styleguide.structure.xml [hunk 4]
```diff
@@ -923,6 +923,128 @@ will be displayed in &lt;emphasis role="bold"&gt;bold&lt;/emphasis&gt;</screen>
...
+  <para>
+   For example, the link
+   <link xlink:href="https://documentation.suse.com/sle-ha/15-SP3/html/SLE-HA-all/article-installation.html#sec-ha-inst-quick-req/"/>
+   points to the <quote>System requirements</quote> for &sle; Server High Availability Extension. To turn this link into an 
+   SP-independent link, you need to identify the different components first:
+  </para>
+  <itemizedlist>
+   <listitem>
+    <para>
+     PRODUCT is <quote>sle-ha</quote>
```
Comment (tomschr) [inline xml/docu_styleguide.structure.xml:993]: Same here:

```suggestion
     PRODUCT is <literal>sle-ha</literal>
```

File: xml/docu_styleguide.structure.xml [hunk 5]
```diff
@@ -923,6 +923,128 @@ will be displayed in &lt;emphasis role="bold"&gt;bold&lt;/emphasis&gt;</screen>
...
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     DEEP_LINK is <filename>/html/SLE-HA-all/article-installation.html#sec-ha-inst-quick-req</filename>
+    </para>
+   </listitem>
+  </itemizedlist>
+  <para>
+   To assemble the link, we can use the following syntax:
```
Comment (tomschr) [inline xml/docu_styleguide.structure.xml:1008]: Maybe we need to rephrase that our current server allows different ways to express such links.

```suggestion
   With the above parts, the redirection rules on our server allows several ways to
   express SP-independent links:
```

File: xml/docu_styleguide.structure.xml [hunk 6]
```diff
@@ -923,6 +923,128 @@ will be displayed in &lt;emphasis role="bold"&gt;bold&lt;/emphasis&gt;</screen>
...
+   <listitem>
+    <para>
+     <uri>https://documentation.suse.com/sle-ha/html/SLE-HA-all/article-installation.html#sec-ha-inst-quick-req</uri>
+    </para>
+    <para>Redirects to the most recent major release, latest SP available</para>
+   </listitem>
+   </itemizedlist>
+  <para>
+   The redirection rules on documentation.suse.com will point to the most recent version.
+  </para>
```
Comment (tomschr) [inline xml/docu_styleguide.structure.xml:1026]: I think this is misleading and was already mentioned above. Would remove it:

```suggestion
```

File: xml/docu_styleguide.structure.xml [hunk 7]
```diff
@@ -923,6 +923,128 @@ will be displayed in &lt;emphasis role="bold"&gt;bold&lt;/emphasis&gt;</screen>
   </para>
  </sect2>
 
+ <sect2 xml:id="sec-link-to-susecom">
+  <title>External links to &suse; documentation</title>
+  <para>
+   In some cases, it makes sense to provide a link which is independent
+   from any Service Pack (SP) number:
```
Comment (tomschr) [inline xml/docu_styleguide.structure.xml:930]: I think we should add some more background info and introduce the term "SP-independent (links)". Maybe something like this?

```suggestion
   Most links in our documentation that goes to <uri>documentation.suse.com</uri>
   refer to a specific product and release. However, sometimes it makes sense to
   <emphasise>not</emphasise> include the SP or even the major release.
   These are so called <emphasis>SP-independent links</emphasis>.
   </para>
   <para>
    The following reasons give you an idea when to use them:
```

File: xml/docu_styleguide.structure.xml [hunk 8]
```diff
@@ -923,6 +923,128 @@ will be displayed in &lt;emphasis role="bold"&gt;bold&lt;/emphasis&gt;</screen>
...
+  </para>
+  <itemizedlist>
+   <listitem>
+    <para>
+     Your linked information is independent from any SP
```
Comment (tomschr) [inline xml/docu_styleguide.structure.xml:935]: Add fullstop at the end:

```suggestion
     Your linked information is independent from any SP.
```

File: xml/docu_styleguide.structure.xml [hunk 9]
```diff
@@ -923,6 +923,128 @@ will be displayed in &lt;emphasis role="bold"&gt;bold&lt;/emphasis&gt;</screen>
...
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     You cannot or do not need to pick a specific SP
```
Comment (tomschr) [inline xml/docu_styleguide.structure.xml:940]:
```suggestion
     You cannot or do not need to pick a specific SP.
```

File: xml/docu_styleguide.structure.xml [hunk 10]
```diff
@@ -923,6 +923,128 @@ will be displayed in &lt;emphasis role="bold"&gt;bold&lt;/emphasis&gt;</screen>
...
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     You want to link to the most recent SP without checking which one it is
```
Comment (tomschr) [inline xml/docu_styleguide.structure.xml:945]:
```suggestion
     You want to link to the most recent SP without checking which one it is.
```

File: xml/docu_styleguide.structure.xml [hunk 11]
```diff
@@ -923,6 +923,128 @@ will be displayed in &lt;emphasis role="bold"&gt;bold&lt;/emphasis&gt;</screen>
...
+    </para>
+   </listitem>
+   <listitem>
+   <para>
+     You want to support SEO where the most prominent SP is more important 
+     than previous, older SPs
```
Comment (tomschr) [inline xml/docu_styleguide.structure.xml:951]:
```suggestion
     You want to support SEO where the most prominent SP is more important 
     than previous, older SPs.
```

**Review by tomschr — CHANGES_REQUESTED** [review]
@dariavladykina Great work! :+1: 
I found some minor issues or provided some ideas.

File: xml/docu_styleguide.structure.xml [hunk 12]
```diff
@@ -923,6 +923,128 @@ will be displayed in &lt;emphasis role="bold"&gt;bold&lt;/emphasis&gt;</screen>
...
+    </para>
+   </listitem>
+  </itemizedlist>
+  <para>
+   Make sure to follow this syntax:
+  </para>
+  <screen>documentation.suse.com/<replaceable>&lt;PRODUCT></replaceable>[/<replaceable>&lt;MAJOR_RELEASE></replaceable>]/<replaceable>&lt;DEEP_LINK>&gt;</replaceable></screen>
```
Comment (dariavladykina) [inline xml/docu_styleguide.structure.xml:963]:
```suggestion
  <screen>documentation.suse.com/<replaceable>&lt;PRODUCT></replaceable>[/<replaceable>&lt;MAJOR_RELEASE></replaceable>]/<replaceable>&lt;DEEP_LINK></replaceable></screen>
```

File: xml/docu_styleguide.structure.xml [hunk 13]
```diff
@@ -923,6 +923,129 @@ will be displayed in &lt;emphasis role="bold"&gt;bold&lt;/emphasis&gt;</screen>
...
+   </listitem>
+   <listitem>
+    <para>
+     DEEP_LINK is <filename>/html/SLE-HA-all/article-installation.html#sec-ha-inst-quick-req</filename>
+    </para>
+   </listitem>
+  </itemizedlist>
+  <para>
+   With the above parts, the redirection rules on our server allows several ways to
+   express SP-independent links:
```
Comment (dariavladykina) [inline xml/docu_styleguide.structure.xml:1014]:
```suggestion
   With the above parts, the redirection rules on our server expressing
   SP-independent links in several ways:
```

File: xml/docu_styleguide.structure.xml [hunk 14]
```diff
@@ -923,6 +923,131 @@ will be displayed in &lt;emphasis role="bold"&gt;bold&lt;/emphasis&gt;</screen>
   </para>
  </sect2>
 
+ <sect2 xml:id="sec-link-to-susecom">
+  <title>External links to &suse; documentation</title>
+  <para>
+   Most links in our documentation that goes to <uri>documentation.suse.com</uri>
+   refer to a specific product and release. However, sometimes it makes sense to
+   <emphasise>not</emphasise> include the SP or even the major release.
+   These are so called <emphasis>SP-independent links</emphasis>.
```
Comment (dariavladykina) [inline xml/docu_styleguide.structure.xml:932]:
```suggestion
   These are so-called <emphasis>SP-independent links</emphasis>.
```

**Review by dariavladykina — COMMENTED** [review]
Implemented some changes.

File: xml/docu_styleguide.structure.xml [hunk 15]
```diff
@@ -923,6 +923,132 @@ will be displayed in &lt;emphasis role="bold"&gt;bold&lt;/emphasis&gt;</screen>
...
+   </listitem>
+   <listitem>
+    <para>
+     DEEP_LINK is <filename>/html/SLE-HA-all/article-installation.html#sec-ha-inst-quick-req</filename>
+    </para>
+   </listitem>
+  </itemizedlist>
+  <para>
+   With the above parts, the redirection rules on our server allows several ways to
+   express SP-independent links:
```
Comment (dariavladykina) [inline xml/docu_styleguide.structure.xml:1015]:
```suggestion
   With the above parts, the redirection rules on our server allow expressing
   SP-independent links in several ways:
```

File: xml/docu_styleguide.structure.xml [hunk 16]
```diff
@@ -923,6 +923,132 @@ will be displayed in &lt;emphasis role="bold"&gt;bold&lt;/emphasis&gt;</screen>
...
+   </listitem>
+   <listitem>
+    <para>
+     DEEP_LINK is <filename>/html/SLE-HA-all/article-installation.html#sec-ha-inst-quick-req</filename>
+    </para>
+   </listitem>
+  </itemizedlist>
+  <para>
+   With the above parts, the redirection rules on our server expressing
+   SP-independent links in several ways:
```
Comment (dariavladykina) [inline xml/docu_styleguide.structure.xml:1015]:
```suggestion
   With the above parts, the redirection rules on our server allow expressing
   SP-independent links in several ways:
```

**Review by dariavladykina — COMMENTED** [review]
Another minor change.

File: xml/docu_styleguide.structure.xml [hunk 17]
```diff
@@ -923,6 +923,132 @@ will be displayed in &lt;emphasis role="bold"&gt;bold&lt;/emphasis&gt;</screen>
   </para>
  </sect2>
 
+ <sect2 xml:id="sec-link-to-susecom">
+  <title>External links to &suse; documentation</title>
+  <para>
+   Most links in our documentation that goes to <uri>documentation.suse.com</uri>
+   refer to a specific product and release. However, sometimes it makes sense to
+   <emphasise>not</emphasise> include the SP or even the major release.
```
Comment (dariavladykina) [inline xml/docu_styleguide.structure.xml:931]:
```suggestion
   <emphasis>not</emphasis> include the SP or even the major release.
```

**Review by dariavladykina — COMMENTED** [review]
Correction of tag

**Pull request merged**